src/lib.rs

```diff
@@ -109,6 +111,8 @@ impl<T: Deserialize> Future for ResponseWrapper<T> {
 
     fn poll(&mut self) -> Poll<Self::Item, Self::Error> {
         let resp = try_ready!(self._poll());
+        #[cfg(feature="gitm_show_responses")]
+        println!("{:#}",String::from_utf8(resp.to_vec()).unwrap());
```

Comment:
nit: ideally format this with a space like `println!("{:#}", String::from_utf8(resp.to_vec()).unwrap());`